### DISCUSSION TITLE
Expose barman-cloud-backup option MIN_CHUNK_SIZE to the operator

### DISCUSSION THREAD

**Discussion by oliverhofmannedb** [cloudnative-pg/cloudnative-pg#3218] [Ideas]
Hi,

with Barman 3.9.0 we got new option to specify MIN_CHUNK_SIZE to barman-cloud-backup. This can be done using '--min-chunk-size' now.

The parameter should be exposed to the operator to leverage this with CNPG backup.

Regards
Oliver

**Comment by Raju-perumal** [comment]
Hi Oliver

Does this parameter help to break the backup instead creating a single data.tar.gz file ?
